File: code/invops.jl
function invops(Bx,By,Bbpx,Bbpy)
    Bxi=kinv(Bx);
    Byi=kinv(By);
    Bbpxi=kinv(Bbpx);
    Bbpyi=kinv(Bbpy);
    return Bxi,Byi,Bbpxi,Bbpyi
end

File: legacy/invops.m
function [Bxi,Byi,Bbpxi,Bbpyi]=invops(Bx,By,Bbpx,Bbpy);

Bxi=inv(Bx);
Byi=inv(By);
Bbpxi=inv(Bbpx);
Bbpyi=inv(Bbpy);
